sierralocal/jsonwriter.py

```diff
@@ -126,15 +127,45 @@ def format_validation_results(self, validated):
         validation_results = [{'level': v[0], 'message': v[1]} for v in validated]
         return validation_results
 
-    def format_drug_resistance(self, scores, gene):
+    def format_drug_resistance(self, scores, sequence_name, gene, ambiguous, names):
         """
         Returns formatted drug resistance and score breakdowns,
         meant for results output.
         @param scores: dict, results of one query from score_alg.score_drugs()
         @param gene: str, gene found in the sequence
+        @param ambiguous: dict, {sequence name: position of NNN NAs}
+        @param names: list, [sequence names]
         @return drug_resistance: dict, one dictionary encoding
         scores and descriptions
         """
+        # remove all ambiguous NNN positions
+        for drug, info in scores.items():
+            new = [[] for i in info[2]]
```

Comment:
it would be clearer to unpack `info` into two named variables instead of indexing into it